openmeter/billing/adapter/invoice.go

```diff
@@ -246,6 +246,7 @@ func (r *adapter) CreateInvoice(ctx context.Context, input billing.CreateInvoice
 		SetNillableDueAt(input.DueAt).
 		SetNillableCustomerTimezone(customer.Timezone).
 		SetNillableIssuedAt(lo.EmptyableToPtr(input.IssuedAt)).
+		SetCustomerSubjectKeys(input.Customer.UsageAttribution.SubjectKeys).
```

Comment:
Just a comment: in the future, we should be able to attribute usage to customers based on different parameters.
Maybe in the future this field would be better to be json store usage attribution like: `
```
.SetCustomerUsageAttribution(input.Customer.UsageAttribution)
```